misk-zookeeper/src/main/kotlin/misk/clustering/zookeeper/ZkLeaseModule.kt

```diff
@@ -22,7 +20,6 @@ import javax.inject.Singleton
 class ZkLeaseModule : KAbstractModule() {
   override fun configure() {
     install(ZkLeaseCommonModule())
-    install(ExecutorServiceModule.withFixedThreadPool(ForZkLease::class, "zk-lease-poller", 1))
```

Comment:
the `RepeatedQueueFactory` uses a `singleThreadedExecutor`, which is the same as fixed size 1